src/braket/aws/aws_session.py

```diff
@@ -825,3 +825,27 @@ def copy_session(
         # Preserve user_agent information
         copied_session._braket_user_agents = self._braket_user_agents
         return copied_session
+
+    def get_full_image_tag(self, image_uri: str) -> str:
+        """
+        Get verbose image tag from image uri.
```

Comment:
Did we validate that this in fact works across all cases?

Reply:
Not fully, still in progress

Reply:
Updated, now it will work as long as there is a tag for the image that includes the python info